client/extensions/woocommerce/woocommerce-services/views/carrier-accounts/index.js

```diff
@@ -69,7 +66,7 @@ CarrierAccounts.propTypes = {
 	siteId: PropTypes.number.isRequired,
 	carriers: PropTypes.arrayOf(
 		PropTypes.shape( {
-			id: PropTypes.string.isRequired,
+			id: PropTypes.string,
```

Comment:
How come `id` is no longer required?

Reply:
It's because the only property always consumed by the frontend is the carrier name, which is called `carrier` and it's the only one required right now.

Reply:
Oh I see. Thanks!